Microsoft.Toolkit.Uwp.UI.Controls.Input/ColorPicker/ColorPicker.cs

```diff
@@ -216,6 +220,7 @@ protected override void OnApplyTemplate()
 
             base.OnApplyTemplate();
             this.UpdateVisualState(false);
+            this.ValidateSelectedPanel();
```

Comment:
There is an issue with validating and updating the selected panel only here. If the `IsColorSpectrumVisible` or `IsColorChannelTextInputVisible` properties are changed after the template is applied the selected tab may become invalid. This is because the base WinUI ColorPicker is currenlty responsible for updating the visual state that actually applies tab visiblity for these two. Only `IsColorPaletteVisible` is managed by the derived toolkit ColorPicker. To fix this:
 1. Need to monitor for property changes on these properties as well, or
 2. Need to connect to VisualState changed events -- there are more than one and this is a bit ugly.

Reply:
This is an existing issue, correct?

Reply:
Yes, I was waiting to see if its considered enough of an issue to fix. The changes now will work as long as the properties are updated in XAML which is by far the largest use case.